src/FontManagerImpl.h

```diff
@@ -0,0 +1,15 @@
+#include "FontDescriptor.h"
+#include "ResultSet.h"
+
+struct FontManagerImpl {
+  FontManagerImpl();
+  ~FontManagerImpl();
+
```

Comment:
I think it's a good practice to delete (`= delete`) copy/move constructor, and copy/move assignment, this way if they are called in the code and that's not the intention, we'll catch it at compile time.
```suggestion
  FontManagerImpl (const FontManagerImpl&) = delete;
  FontManagerImpl& operator= (const FontManagerImpl&) = delete;
  FontManagerImpl (FontManagerImpl&&) = delete;
  FontManagerImpl& operator= (FontManagerImpl&&) = delete;
```

Reply:
damn. nice catch. I always forget about the copy/move constructors